<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkConfigDialog</class>
 <widget class="QDialog" name="NetworkConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="networkConfigLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Network Configuration</string>
   </property>
  </widget>
  <widget class="Line" name="footerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="headerSeparator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="visionGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="visionIpForm">
     <item row="0" column="0">
      <widget class="QLabel" name="visionIpLabel">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="visionIpInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="visionPortForm">
     <item row="0" column="0">
      <widget class="QLabel" name="visionPortLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="visionPortInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="visionFormLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vision:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="refereeGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="refereeFormLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Referee:</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="refereePortForm">
     <item row="0" column="0">
      <widget class="QLabel" name="refereePortLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="refereePortInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="refereeIpForm">
     <item row="0" column="0">
      <widget class="QLabel" name="refereeIpLabel">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="refereeIpInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="formLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="aiIpForm">
     <item row="0" column="0">
      <widget class="QLabel" name="aiIpLabel">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="aiIpInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="aiRPortForm">
     <item row="0" column="0">
      <widget class="QLabel" name="aiRPortLabel">
       <property name="text">
        <string>R:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="aiRPortInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="aiSPortForm">
     <item row="0" column="0">
      <widget class="QLabel" name="aiSPortLabel">
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="aiSPortInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="aiFormLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AI:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="defaultsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Defaults</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>refereeIpInput</tabstop>
  <tabstop>refereePortInput</tabstop>
  <tabstop>aiIpInput</tabstop>
  <tabstop>aiSPortInput</tabstop>
  <tabstop>aiRPortInput</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
